imports/plugins/included/marketplace/server/methods.js

```diff
@@ -13,8 +13,8 @@ Meteor.methods({
    * @returns {Object|null} - The user hash if found, null otherwise
```

Comment:
These methods ("marketplace/getSeller", "marketplace/getSellerShop") are of no use anymore. (They are replaced by getSeller ,getSellerShop ) of lib/api/core.js